### ISSUE TITLE
Hidden files may appear in Quick Switcher as recently opened notes

### ISSUE THREAD

**Issue by dbogdanov** [catvatar/Obsidian-NSFW-Plugin#1]
Currently, hidden files appear in the Quick Switcher if they are among the recently opened notes. I'm not sure if there is a way to modify the history of recently opened notes to prevent this.

**Comment by catvatar** [comment]
Thanks for your feedback I'll look into it

**Comment by catvatar** [comment]
Hidden files disappear from the Quick Switcher when NSFW is toggled. 
There is a different issue that I don't see a straight forward solution to. If a note is referenced by another note then it will be visible under the Quick Switcher as a not yet created file.
Does this describe your issue? If so a quick fix is to also set the notes referencing NSFW notes as NSFW.

**Comment by dbogdanov** [comment]
No, the issue is different: When pressing Ctrl+O, Quick Switcher shows a history of recently opened notes until a user types in their search. This history may contain NSFW notes.

Additionally, I just tested that notes which are referenced somewhere but not yet created do not appear in Quick Switcher, so this is not a problem.

**Comment by catvatar** [comment]
Hi, could you provide the steps to recreate this bug on the sandbox vault with only this plugin installed.
